### ISSUE TITLE
FR: Option to disable in Markdown links

### ISSUE THREAD

**Issue by chrisgrieser** [Automattic/harper#104]
Could there be an option to disable spellchecking the content of markdown links? There are many cases, where doing so just results in tons of false positives, e.g. in the list of GitHub repos.

ltex does [have a setting](https://valentjn.github.io/ltex/settings.html#ltexmarkdownnodes) to configure whether code blocks, inline code, or links should be spellchecked, and it would be useful if harper had such a setting as well.

![Pasted image 2024-08-07 at 12 18 36@2x](https://github.com/user-attachments/assets/d4a98922-73aa-4ea5-8d06-f7cf4c398b7e)

**Comment by elijah-potter** [comment]
Absolutely! I've got some changes stashed that I want to polish first, but then it'll be top priority.

**Comment by elijah-potter** [comment]
> but then it'll be top priority.

:billed_cap:

Sorry about that. @grantlemons, could you take this up?

**Comment by chrisgrieser** [comment]
Just stumbled upon this in the docs and noticed that the issue is now fixed. 🥳 

```lua
lspconfig.harper_ls.setup {
  settings = {
    ["harper-ls"] = {
      markdown = {
        ignore_link_title = true,
      }
    }
  },
}
```
